operations/mimir-mixin/config.libsonnet

```diff
@@ -232,7 +232,10 @@
             container_memory_rss{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"}
               /
             ( container_spec_memory_limit_bytes{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"} > 0 )
-          ) > %(allocationpercent)s
+          ) *
+          # Make sure that the pods are really all Mimir pods.
+          on(namespace, pod, container) group_left group(cortex_build_info{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"}) by (namespace, pod, container)
```

Comment:
I think it will only work if we make sure there are only the labels `(namespace, pod, container)` on first part of the expression, currently there are labels like `job, id, name, image` on the metric. 

I done it this way to not change the label set produced

Reply:
Having another think: It should do what we want without a 1-many join like that:

https://github.com/grafana/mimir/pull/5045/commits/464f7504e6f407131bc4f220f6005592ea37bebe